ingest/ingester.py

```diff
@@ -33,6 +34,7 @@ def __init__(self, collection_name: str, content_folder: str, vectordb_folder: s
         self.vecdb_type = settings.VECDB_TYPE if vecdb_type is None else vecdb_type
         self.chunk_size = settings.CHUNK_SIZE if chunk_size is None else chunk_size
         self.chunk_overlap = settings.CHUNK_OVERLAP if chunk_overlap is None else chunk_overlap
+        self.local_api_url = settings.LOCAL_API_URL if local_api_url is None and settings.LOCAL_API_URL is not None else local_api_url
```

Comment:
Hi @lrvl , @Sennaa , this parameter is still missing in settings.py
Shouldn't it be added there?